src/sentry/api/endpoints/organization_teams.py

```diff
@@ -116,6 +116,12 @@ def get(self, request, organization):
             else team_serializers.TeamSerializer
         )
 
+        get_all_teams = request.GET.get("all_teams") == "1"
+
+        if get_all_teams:
+            queryset = queryset.order_by("slug")
+            return Response(serialize(list(queryset), request.user, serializer()))
```

Comment:
Couldn't we also run into slow responses for very large orgs. Should we be loading via pagination, or updating frontend components to use autocomplete/typeahead based loading?

Reply:
yea that is fair, the only reason I wanted to do this is because multiple users have messaged support with incomplete team lists on the create project page. Also, any other frontend components that exist in the lightweight organization routes which access the TeamStore would have an incomplete set of teams. So far this isn't a huge problem because it seems like there are not that many orgs with over 100 teams. 

I'm definitely down to update frontend components to use autocomplete/typeahead loading, but am unsure how long that will take and am just offering this as a quick fix in the meantime so users don't keep running into the same issues. If this PR is not ideal though I can look into doing typeahead based loading.